WebApp/package.json

```diff
@@ -0,0 +1,247 @@
+{
+  "name": "dap.ps-frontend",
+  "version": "1.0.0",
+  "description": "",
+  "keywords": [
+    "React",
+    "Redux",
+    "Typescript"
+  ],
+  "author": "Ryan Noble",
+  "license": "MIT",
+  "engines": {
+    "node": ">=8.0",
+    "yarn": ">=1.9.4"
+  },
+  "scripts": {
+    "analyze:clean": "cross-env rimraf stats.json",
+    "preanalyze": "yarn analyze:clean",
+    "analyze": "node ./internals/scripts/analyze.js",
+    "extract-intl": "node ./internals/scripts/extract-intl.js",
+    "npmcheckversion": "node ./internals/scripts/npmcheckversion.js",
+    "preinstall": "yarn npmcheckversion",
+    "prebuild": "yarn build:clean",
+    "build": "cross-env NODE_ENV=production webpack --config internals/webpack/webpack.prod.babel.js --color -p --progress --hide-modules --display-optimization-bailout",
+    "build:clean": "rimraf ./build",
```

Comment:
This also assumes that `rimraf` is avaialble in `PATH`. Use `./node_modules/.bin/rimraf`.

Reply:
Noted, will amend